karma.py

```diff
@@ -1,4 +1,5 @@
 from brutal.core.plugin import BotPlugin, cmd, match
+from brutal.core.utils import change_cmd
```

Comment:
@mrshu since this requires brutal-ng to work, should we also mention it in README or somewhere?